ibis/common/tests/test_patterns.py

```diff
@@ -97,6 +104,30 @@ def __eq__(self, other):
         return self.__class__ == other.__class__ and self.min == other.min
 
 
+x = Variable("x")
+y = Variable("y")
+z = Variable("z")
+
+
+def test_always():
+    p = Always()
```

Comment:
`Ident` or `Identity`?

Reply:
Then how would you call `Never()`?

Reply:
Reconsidering this, I think `Always` is fine.